### DISCUSSION TITLE
定时任务支持设定工作文件集

### DISCUSSION THREAD

**Discussion by kingname** [whyour/qinglong#2131] [Q&A]
有时候某一些文件需要在特定文件夹下面运行，但是目前创建定时任务的地方无法指定工作文件夹。

使用 cd xxx/yyy && python test.py  这种写法也不行，它依然会把工作文件就设定为/ql

**Comment by whyour** [comment]
任务运行前执行 cd xxx/yyy
任务命令使用 task xxx/yyy/test.py 不行吗
